automated install/basic-install.sh

```diff
@@ -1388,8 +1441,17 @@ install_dependent_packages() {
             printf "  %b Error: Unable to find Pi-hole dependency package.\\n" "${COL_RED}"
             return 1
         fi
-
-    # If neither apt-get or yum/dnf package managers were found
+    # Install Alpine packages
+    elif is_command apk; then
+        sed -i 's:#\(.*/community\):\1:' /etc/apk/repositories &&
+            printf "%b  %b Enabled 'community' repo.." "${OVER}" "${INFO}"
```

Comment:
Should this message be suppressed/changed if the community repo was already enabled by the user?

By default sed returns success even if it made no changes (it only returns differently if there is an error eg unable to reading or write the file), so anything guarded by the && will still be executed even if no changes were made.

Reply:
I did consider that, but decided it wasn't worth the added complexity to add a redundant check. Thinking more about it though, I am currently just relying on the install command to fail if `sed` fails, but it really should show a dedicated error if the community repo can't be enabled. Since only the error case is actually useful, I'll wrap the `sed` call in a condition and only show a message if it fails.